QUEST_LV_0100.tsv

```diff
@@ -8116,7 +8116,7 @@ QUEST_LV_0100_20150918_008111	You are so determined unlike other guys nowadays.{
 QUEST_LV_0100_20150918_008112	$During your travels you might discover Goddess Statues scattered all over the kingdom.{nl}Worship these statues and you will certainly receive their blessings.
 QUEST_LV_0100_20150918_008113	Press {img F10 40 40} to open the Help menu for more information.
 QUEST_LV_0100_20150918_008114	The Writings on the Wall
-QUEST_LV_0100_20150918_008115	Which factors are decisive for determining outcomes of battles?{nl}If you just continue to attack thoughtlessly, then you may get attacked instead.
+QUEST_LV_0100_20150918_008115	Which factors are decisive for determining the outcome of battle?{nl}If you just continue to attack thoughtlessly, then you may get attacked instead.
```

Comment:
Either work.  Saying "the outcome of battle" is a little more abstract, which leans towards the tone of this sentence a bit better, I think.